diff --git usr/share/applications/wifiman-desktop.desktop usr/share/applications/wifiman-desktop.desktop
index f8b0fe9..1331e7d 100644
--- usr/share/applications/wifiman-desktop.desktop
+++ usr/share/applications/wifiman-desktop.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=WiFiman Desktop
-Exec="/opt/WiFiman Desktop/wifiman-desktop" %U
+Exec="/opt/wifiman-desktop/wifiman-desktop" %U
 Terminal=false
 Type=Application
 Icon=wifiman-desktop
