diff --git opt/WiFiman Desktop/service/wifiman-desktop.service opt/WiFiman Desktop/service/wifiman-desktop.service
index 7e757d6..c0fb85c 100644
--- opt/WiFiman Desktop/service/wifiman-desktop.service	
+++ opt/WiFiman Desktop/service/wifiman-desktop.service	
@@ -6,7 +6,7 @@ After=network.target
 User=root
 Restart=always
 RestartSec=10
-ExecStart="/opt/WiFiman Desktop/service/wifiman-desktopd"
+ExecStart=/opt/wifiman-desktop/service/wifiman-desktopd
 
 [Install]
 WantedBy=multi-user.target
\ No newline at end of file
